file-upload/file-upload-java-sbt/fileupload-impl/src/main/resources/routes

```diff
@@ -0,0 +1 @@
+POST    /api/files      com.example.play.controllers.FileUploadController.upload()
```

Comment:
Does this require:

```
->  /  LagomServiceRouter
```
?

(as in https://github.com/lagom/lagom-recipes/blob/master/file-upload/file-upload-scala-sbt/fileupload-impl/src/main/resources/routes)

Reply:
I guess it's not necessary since the tests pass. 🤔 

Is this behaving differently in java and scala or is the scala sample wrong?

Reply:
Since this example uses 1.5, we have to add only ADDITIONAL routes. May be it's also possible to write router manually instead of using generated by the Play one.

Reply:
I wrote these comments before realising this recipe uses `additionalRoutes` API in the `Module` to obtain the final `Router`.